### PR TITLE
Merge changes into release/2.1

### PR DESCRIPTION
**Pull request by wtgodbe**
CC @mmitche

### CODE REVIEW

**Comment by wtgodbe** [comment]
CC @weshaggard is the change to System.Runtime.CompilerServices.Unsafe in packageindex.json expected? I got this by merging in changes from VSTS.

**Comment by wtgodbe** [comment]
Note to self - don't squash this

File: pkg/Microsoft.Private.PackageBaseline/packageIndex.json [hunk 1]
```diff
@@ -3607,6 +3607,9 @@
         "4.5.0",
         "4.5.1"
       ],
+      "InboxOn": {
+        "uap10.0.16300": "4.0.4.0"
```
Comment (weshaggard) [inline pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: This was removed as part of https://github.com/dotnet/corefx/pull/29970 so I don't think it should be added back.

**Comment by stephentoub** [comment]
> Note to self - don't squash this

How come?

**Comment by stephentoub** [comment]
Ok. That shouldn't require six additional commits.

**Comment by wtgodbe** [comment]
@stephentoub do you suggest rebasing & squashing the rest of the commits?

**Comment by mmitche** [comment]
You could only squash after the commit you want.

**Comment by wtgodbe** [comment]
This is now in the correct state, we need to maintain the commit chain to get the correct commit hashes.

**Comment by weshaggard** [comment]
>I obviously have a gap in my git knowledge here... what does it mean for the commit that resulted from merging this to be empty?

I believe it was a result of a cherry-pick of the same exact change but a different commit. When merged there were no changes just the merges in order to have a matching sha for what we built from.

**Comment by stephentoub** [comment]
Ok, thanks.

**Pull request merged**